### ISSUE TITLE
PersistGate with nested persists

### ISSUE THREAD

**Issue by DevNamedZed** [rt2zz/redux-persist#575]
Is it possible to use the PersistGate  with nested persists?

**Comment by rt2zz** [comment]
Yes, assuming you mean nested persistReducer's, that works out of the box.

The basic logic is this:
1. the persistor is considered to be "bootstrapped" once all persistoids that were present upon the initial render are rehydrated.
2. PersistGate lifts once the persistor is bootstrapped

The idea is that on initial render most apps will want to wait for all state to rehydrate, *but* if a reducer is later added (e.g. via code splitting) even though a new rehydrate is pending, the gate should remain lifted.
